commons/src/config/preamble.rs

```diff
@@ -111,15 +111,25 @@ pub struct Citedata {
     #[serde(default)]
     pub references: HashSet<PathBuf>,
     /// Optional files containing locale information to render citations.
-    #[arg(long, value_parser = parse_to_locale_pathbuf_hashset, required = false, default_value = "")]
+    #[clap(skip)]
     #[serde(with = "locale::serde::hashmap", default)]
     pub citation_locales: HashMap<Locale, PathBuf>,
+
+    #[arg(long = "csl-locale", value_parser = parse_locale_path_buf, required = false, default_value = "")]
+    #[serde(skip)]
+    pub csl_locales: Vec<(Locale, PathBuf)>,
 }
 
 impl ConfigFns for Citedata {
     fn merge(&mut self, other: Self) {
         self.style.replace_none(other.style);
         self.references.extend(other.references);
+        for locale_path_buf in self.csl_locales.clone() {
```

Comment:
```suggestion
        for (locale, pathbuf) in self.csl_locales.clone() {
```

you can destructure tuples directly